db/seeds/source_types.yml

```diff
@@ -74,6 +74,81 @@ amazon:
         :initialValue: aws
   :vendor: Amazon
   :icon_url: "/apps/chrome/assets/images/partners-icons/aws.svg"
+amazon-local:
```

Comment:
@dccurtis I agree with @slemrmartin in regards to not needing `-local` here if we use a `SUPPORTED_TYPES` pattern. Here are more examples around how that pattern was applied: https://github.com/RedHatInsights/topological_inventory-orchestrator/pull/79

Reply:
Thanks @syncrou !  I haven't forgotten about this.  Hope to have some updates to this branch this week.